tests/unittests/ImageIO/ImageIOTest.mm

```diff
@@ -691,7 +692,7 @@ static CFURLRef getURLRefForOutFile(const wchar_t* filename) {
     CFRelease(imageSource);
 }
 
-TEST(ImageIO, DestinationTest) {
+DISABLED_TEST(ImageIO, DestinationTest) {
```

Comment:
These are being taken care of it in 16/11.
It's part of the re-integration tasks (which will be created)

---
In reply to: [85251230](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r85251230) [](ancestors = 85251230)